queries.yaml

```diff
@@ -1841,6 +1841,38 @@ post_process:
       # need to be a large tolerance.
       simplify_tolerance: 1.0
 
+  # drop labels on roads which are too short to render.
+  - fn: vectordatasource.transform.drop_names_on_short_boundaries
+    params:
+      source_layer: roads
+      start_zoom: 8
+      end_zoom: 15
+      # pixels (256px nominal) to require per letter in the name. if lines are shorter than this,
+      # the name gets dropped. name pairs (e.g: :left and :right) get dropped if either are longer
+      # than pixels_per_letter. larger values mean fewer lines are eligible for labelling.
+      pixels_per_letter: 11
```

Comment:
Generally this yields a 7% decrease in tile size, woot!

Let's do another build with setting of 8 instead of 11 to see file size versus dropped names for aesthetics.